Sources/BaggageLogging/Baggage+Logger.swift

```diff
@@ -0,0 +1,19 @@
+import Baggage
+import Logging
+
+extension BaggageContext {
+    public var logger: Logger {
+        var logger = self[BaseLoggerKey.self] ?? Logger(label: "BaggageContext")
+        // TODO: Filtering out a specific key to get rid of the logger seems very strange
+        for (key, value) in baggageItems where key.keyType != BaseLoggerKey.self {
```

Comment:
ticket or some writeup about this problem somewhere?

What if we go down the route of "context is a bag of ANYTHING" then there could be random stuff in here including functions etc which we'd also not want to make into a metadata. Would we need to make some other type of key for "Actual metadata" and "terrible closure" and such things would have to be aware of it?

That's quite a problem if we allow random stuff in contexts other than "just data"... What would be another way we can solve this?